
<!DOCTYPE html>
<html>
<head>
  <title>MSEL-Molecular Systems Engineering Lab</title>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">

  <!-- <link rel="stylesheet" href="css/all.css"> -->
  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">

  <link rel="stylesheet" type="text/css" href="css/style.css">
  <link rel="stylesheet" type="text/css" href="css/html5reset.css">
  <link href="https://fonts.googleapis.com/css?family=Raleway" rel="stylesheet">
    <link rel="stylesheet" type="text/css" href="css/responsive.css">

  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
</head>
<body>
  <div id="wrapper">
    <div class="header">
      <div class="showcase">
        <div class="logo">
          <img src="img/emp.gif" class="logo-center">
        </div>
        <div class="title">
          <h1>Molecular Systems Engineering Lab</h1>
          <h3>(Genomic Energy of Bioresources)</h3>
          <h2 class='dept'>Department of bioinformatics</h3>
        </div>
      </div>
      <div class = "nav-bar">
<div class="topnav" id="myTopnav">
  <a  href="home.html" class="btn ">Home</a>
  <a href="biography.html" class="btn">biography</a>
  <a href="publications.html" class="btn">publications</a>
  <a href="researchers.html" class="btn">Researchers</a>
  <a href="facility.html" class="btn">Facility</a>
  <a href="resources.html" class="btn active">Resources</a>
  <a href="javascript:void(0);" class="icon" onclick="myFunction()">
              <i class="fa fa-bars"></i>
              </a>
</div>
        </div>
      </div>
 <div class="content">
            <a href="CPSIR/home.pl"><h3>CPSIR-CM: Cyanobacterial Protein Structure Information Resource for C1Metabolisms</h3></a>
            <p>Cyanobacteria are capable of degrading both naturally occurring aliphatic and aromatic hydrocarbons. In this study, database structure is organized for completed cyanobacterial genomes comprising information to 9 metabolic pathways. Protein structural information for starvation induced proteins, C1 transporters and assimilation pathways to methane, aliphatic, nitrogenated, halogenated and sulfated C1 compounds have incorporated in this database. Structural information for more than 500 functional proteins has included individually describing their features at primary, secondary and tertiary levels. Along with NCBI accession we have also given unique number to retrieve enzyme of interest in specified pathways of genus alone. Protein 3D models of each entry are implemented in this resource in PDB format that would be supporting for molecular modeling, dynamics and docking studies on understanding the structure-function relationships. NCBI accession, pathway, names of enzymes and organism are assigned as search forms in a search engine supported by this database.<a href="CPSIR/home.pl" target="_blank"> Click Here</a></p>
            
            <h3>GSMMM Portel: Genome-Scale Microbial Metabolic Models Portel</h3>
            <p>A central goal of molecular systems biology is the elucidation of cell function and physiology through the integrated use of broad based genomic and physiological data. Genome-scale models of microbial organisms, comprising different levels of information, are primarily on the stoichiometry of many different reactions. Some information about metabolic regulation could offer a suitable platform for developing systems level tools for analyzing and engineering metabolism.  The in silico model has also been used to predict the metabolic adaptation to several environmental perturbations. These predictions are currently being compared with experimental data, including physiological measurements and global gene expression data, to achieve the goal of improvement in our understanding on the metabolism of microorganisms under non-optimal but environmentally relevant conditions.  In our lab, we are constructing genome-scale models for Methanobrevibacter ruminantium M1, Methanosarcina mazei GO1, Methanothermobacter thermautotrophicus delta H,  Methanothermococcus okinawensis, Clostridium sticklandii HF, Clostridium botulinum type A and type B.</p>
            </p>
            <div class="group-table">
              <div>
                <h1 align="center">Structural genes</h1>
                <br>
<table >
<tbody><tr style=" background:#00356b; color:#FFFFFF;">
<th width="20%">Accession</th>
<th width="15%">Gene</th>
<th width="65">Organism</th>
</tr>
<tr>
<td>EU826071</td>
<td>16S rRNA </td>
<td><i>Ochrobactrum anthropi strain SRK5</i></td>
</tr>

<tr>
<td>EU826070</td>
<td>16S rRNA </td>
<td><i>Ignatzschineria larvae strain SRK4</i></td>
</tr>
    



<tr>
<td>EU826069</td>
<td>16S rRNA </td>
<td><i>Ochrobactrum cytisi strain SRK3</i></td>
</tr>

<tr>
<td>EU513181</td>
<td>16S rRNA </td>
<td><i>Photorhabdus luminescens subsp. laumondii strain SRK 2</i></td>
</tr>

<tr>
<td>EU513180</td>
<td>16S rRNA </td>
<td><i>Xenorhabdus poinarii strain SRK 1</i></td>
</tr>

<tr>
<td>EU159696</td>
<td>16S rRNA </td>
<td><i>Leptospira interrogans serovar Bataviae strain Swart</i></td>
</tr>

<tr>
<td>EU159695</td>
<td>16S rRNA </td>
<td><i>Leptospira santarosai serovar Sarmin strain CZ390</i></td>
</tr>

<tr>
<td>EU159694</td>
<td>16S rRNA </td>
<td><i>Leptospira borgpetersenii serovar Javanica strain Poi</i></td>
</tr>

<tr>
<td>EU159693</td>
<td>16S rRNA </td>
<td><i>Leptospira interrogans serovar Autumnalis strain N2</i></td>
</tr>

<tr>
<td>EU159692</td>
<td>16S rRNA </td>
<td><i>Leptospira biflexa serovar Patoc strain Patoc 1</i></td>
</tr>

<tr>
<td>EU159691</td>
<td>16S rRNA </td>
<td><i>Leptospira interrogans serovar Hardjo strain Hardjoprajitno</i></td>
</tr>

<tr>
<td>EU159690</td>
<td>16S rRNA </td>
<td><i>Leptospira noguchii serovar Louisiana strain LSU1945</i></td>
</tr>



<tr>
<td>EU159689</td>
<td>16S rRNA </td>
<td><i>Leptospira sp. G6</i></td>
</tr>

<tr>
<td>EU159688</td>
<td>16S rRNA </td>
<td><i>Leptospira sp. G3</i></td>
</tr>

<tr>
<td>EU159687</td>
<td>16S rRNA </td>
<td><i>Leptospira interrogans serovar Australis strain Ballico</i></td>
</tr>

<tr>
<td>EU159686</td>
<td>16S rRNA </td>
<td><i>Leptospira borgpetersenii serovar Javanica strain R1R</i></td>
</tr>

<tr>
<td>EU159685</td>
<td>16S rRNA </td>
<td><i>Leptospira borgpetersenii serovar Javanica strain R1L</i></td>
</tr>
</tbody></table>
              </div>
              

<div>
  <h1 align="center">Functional genes</h1>
  <br>
  <table>
<tbody>
  <tr style="background:#00356b; color:#FFFFFF;">
<th width="20%">Accession</th>
<th width="15%">Gene</th>
<th width="65">Organism</th>
</tr><tr>
<td>EU091296</td>
<td>OmpL41</td>
<td><i>Leptospira interrogans serovar Grippotyphosa</i></td>
</tr>
<tr><td>EU091295</td>
<td>OmpL41</td>
<td><i>Leptospira interrogans serovar Autumnalis</i></td>
</tr>

<tr><td>EU091294</td>
<td>OmpLL1</td>
<td><i>Leptospira interrogans serovar Autumnalis</i></td>
</tr>
<tr>
<td>EU091293</td>
<td>OmpLL1</td>
<td><i>Leptospira interrogans serovar Grippotyphosa</i></td>
</tr>

<tr>
<td>ACB12047</td>
<td>OmpL32</td>
<td><i>Leptospira interrogans serovar Autumnalis</i></td>
</tr>

<tr>
<td>ACB12046</td>
<td>OmpL32</td>
<td><i>Leptospira borgpetersenii serovar Javanica</i></td>
</tr>

<tr>
<td>ACB12045</td>
<td>OmpL32</td>
<td><i>Leptospira borgpetersenii serovar Javanica</i></td>
</tr>


<tr>
<td>ABX83722</td>
<td>OmpL32</td>
<td><i>Leptospira borgpetersenii serovar Javanica</i></td>
</tr>

<tr>
<td>ABX83723</td>
<td>OmpL32</td>
<td><i>Leptospira interrogans serovar Autumnalis</i></td>
</tr>
<tr>
<td>EU526391</td>
<td>OmpL32</td>
<td><i>Leptospira interrogans serovar Autumnalis strain N2</i></td>
</tr>

<tr>
<td>EU526390</td>
<td>OmpL32</td>
<td><i>Leptospira borgpetersenii serovar Javanica strain R1L</i></td>
</tr>

<tr>
<td>EU526389</td>
<td>OmpL32</td>
<td><i>Leptospira borgpetersenii serovar Javanica strain R1R</i></td>
</tr>

<tr>
<td>EU293442</td>
<td>OmpL32</td>
<td><i>Leptospira borgpetersenii serovar Javanica strain R1L</i></td>
</tr>

<tr>
<td>EU293441</td>
<td>OmpL32</td>
<td><i>Leptospira borgpetersenii serovar Javanica strain R1L</i></td>
</tr>


<tr>
<td>EU293443</td>
<td>OmpL32</td>
<td><i>Leptospira interrogans serovar Autumnalis strain N2</i></td>
</tr>

<tr>
<td>FJ434139</td>
<td>SecY</td>
<td>Uncultured Leptospira sp.</td>
</tr>

<tr>
<td>FJ434138</td>
<td>SecY</td>
<td>Uncultured Leptospira sp.</td>
</tr>

<tr>
<td>FJ434137</td>
<td>SecY</td>
<td>Uncultured Leptospira sp.</td>
</tr>

<tr>
<td>FJ755847</td>
<td>Collagen type II  </td>
<td><i>Gallus gallus.</i></td>
</tr>
</tbody></table>
</div>

            </div>
         </div>



    <div class="footer ">
        <div class="footer-container ">
          <div class="footer-col-2 ">
            
            <div itemprop="address" itemscope=" " itemtype="http://schema.org/PostalAddress ">
              <h1 align="center"><u>Contact us</u></h1>
              <p><br><span itemprop="name"><strong>Prof. P. CHELLAPANDI</strong></span><br><span itemprop="name ">Department of Bioinformatics</span><br><span itemprop="addressRegion ">Bharathidasan University, Tiruchirappalli-620 024</span><br><span itemprop="addressLocality ">Tamil nadu, INDIA</span>
              </p>
            </div>
            
            <div>
              <p> <br><a href="tel:+13193351050 " itemprop="telephone "><i aria-hidden="true " class="fa fa-phone "></i> 989-410-9534</a><br>   <a href="mailto:pchellapandi@gmail.com " itemprop="email ">pchellapandi@gmail.com</a> <br><br></p>
            </div>
          </div>

          <div class="credit ">
            Website Developed by R Prakash & R Prathiviraj
          </div>
        </div>
      </div>
    </div>


    
  </div>

<script>
function myFunction() {
  var x = document.getElementById("myTopnav");
  if (x.className === "topnav") {
    x.className += " responsive";
  } else {
    x.className = "topnav";
  }
}
</script>

</body>
</html>